include/PositionLine.h

```diff
@@ -38,8 +38,10 @@ class PositionLine : public QWidget
 	Q_OBJECT
 	Q_PROPERTY(bool tailGradient MEMBER m_hasTailGradient)
 	Q_PROPERTY(QColor lineColor MEMBER m_lineColor)
+	Q_PROPERTY(QColor recordingColor MEMBER m_recordingColor)
 public:
 	PositionLine(QWidget* parent, Song::PlayMode playMode);
+	bool isRecording = false;
```

Comment:
This should be private and named `m_isRecording`.

Then you can have a getter and setter like this:
```cpp
bool isRecording() const { return m_isRecording; }
void setRecording(bool on) { m_isRecording = on; }
```